CMakeLists.txt

```diff
@@ -1,13 +1,19 @@
 cmake_minimum_required(VERSION 3.10)
-project(vikunja)
+project(vikunja
+  VERSION      0.1.0
+  DESCRIPTION  "Primitives for Alpaka"
+  HOMEPAGE_URL "https://github.com/ComputationalRadiationPhysics/vikunja"
+  LANGUAGES    CXX)
 
 set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra")
 option(VIKUNJA_REDUCE_COMPARING_BENCHMARKS_ENABLE "Should the reduce integration test run some comparison benchmarks?" OFF)
 option(VIKUNJA_TRANSFORM_COMPARING_BENCHMARKS_ENABLE "Should the transform integration test run some comparison benchmarks?" OFF)
 
-list(APPEND CMAKE_MODULE_PATH "${ALPAKA_ROOT}")
-set(ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLED ON)
+if(${CMAKE_PROJECT_NAME} STREQUAL ${PROJECT_NAME})
+  list(APPEND CMAKE_MODULE_PATH "${ALPAKA_ROOT}")
+  option(ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLE "enable alpaka serial accelerator" ON)
```

Comment:
@sbastrakov and @psychocoderHPC: is this how we expected users of alpaka to deal with our disabling of all backends by default? Because I was surprised back then when no backend was enabled by default in alpaka and someone of you told me, it's better that the user explicitely enables one. So vikunja is now enabling a backend automatic again. Is that good or bad now? I really do not know :S

Reply:
This is a good question. This line enables alpaka sequential accelerator that should be available everywhere and so it's safe to do it. However this accelerator should never be used for "actual" (parallel) alpaka-based computing.

Reply:
I think, we should avoid it in general. But in this special case, I recognize, that the place is simply wrong. The backend is necessary for the tests and example, so I should move it to the CMakeLists.txt of it.

Reply:
I forgot, that the acc cannot be chosen after `find_package`. Now I enable the acc in the top level CMakeLists.txt, if the build the of examples and/or tests are enabled and `ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLE` was not set before.